modules/asset-manager-impl/src/main/java/org/opencastproject/assetmanager/impl/endpoint/AbstractTieredStorageAssetManagerRestEndpoint.java

```diff
@@ -51,6 +54,7 @@
 
 public abstract class AbstractTieredStorageAssetManagerRestEndpoint extends AbstractAssetManagerRestEndpoint {
 
+  private DateFormat sdf = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss'Z'");
```

Comment:
DateFormat is not thread safe so it should be a local variable.

Reply:
Ah, right.